docs/cloud-tutorials/monitoring-dashboard.md

```diff
@@ -1,41 +1,51 @@
 ---
 sidebar_position: 6
-title: Monitoring Dashboard
-description: Learn how to monitor DBOS Cloud applications
+title: Monitoring Your Applications
 ---
 
-In this guide, you'll learn how to use the monitoring dashboard to visualize the call history and performance of applications in DBOS Cloud.
+The [DBOS Cloud Console](https://console.dbos.dev) provides several tools to monitor your applications.
 
-### Preliminaries
+### Logs
 
-In order to launch the dashboard you must first deploy an application, as described in [quickstart](../quickstart.md). Having done so, from your application directory, run:
+You can view your application's logs from your application's cloud console page.
+Logs are paginated and ordered chronologically.
 
-```
-dbos-cloud dashboard url
-```
+![Logs](./assets/cc-logs.png)
 
-This command will output the URL of your dashboard. Log in with the same credentials you used to register your app. 
+### Traces
 
-### Time Selection
+You can view traces for all your applications from the cloud console [traces page](https://console.dbos.dev/traces).
+You can filter traces by application, time, operation, type, and status.
+Traces are sorted chronologically and displayed hierarchically.
+You can click on a trace or span to see detailed information about it.
 
-In the top-right corner, the dashboard provides a time selector, defaulting to the last hour. You can change this setting to navigate to a different window of time. All of the panels are filtered for the selected time interval.
+![Logs](./assets/cc-traces.png)
+
+
+### Grafana Dashboard
+
+You can launch a Grafana dashboard for DBOS Cloud from the cloud console [dashboard page](https://console.dbos.dev/dashboard).
+
+#### Time Selection
+
+In the top-right corner, the Grafana dashboard provides a time selector, defaulting to the last hour. You can change this setting to navigate to a different window of time. All of the panels are filtered for the selected time interval.
 ![Time picker](./assets/time_picker.png)
 
 Under the time selector you can find time series for active CPU milliseconds used by your apps and the counts of logs and traces generated by your applications, summarized for every minute. These panes have a matched time axis. You can click and drag across an interesting region in the series to "zoom in". 
 ![Series](./assets/timeseries.png)
 
 This will update the time selector and, therefore, all other panels. You can then use the time selector to "zoom back out" or use the `<` and `>` buttons to move backwards and forwards in time.
 
-### Logs and Traces
+#### Grafana Logs and Traces
```

Comment:
I don't see a reason why we need to include the word "Grafana". Sure it's obvious to anyone who's seen Grafana before but does it help us to say it?